components/button/demo/icon.md

```diff
@@ -1,25 +1,29 @@
 ---
 order: 1
-title: 图标按钮
+title:
+  zh-CN: 图标按钮
+  en-US: Icon
 ---
 
+## zh-CN
+
 `Button` 内可以嵌套图标，图标可以放在文字前、后，也可以单独存在。
 
+## en-US
+
+`Button` works fine with `Icon`, just set `icon` property or put `Icon` component in `Button`
```

Comment:
`Button` components can contain an `Icon`. This is done by setting the `icon` attribute or placing an `Icon` component within the `Button`

If you want specific control over the positioning and placement of the `Icon`, then that should be done by placing the `Icon` component within the `Button` rather than using the `icon` attribute.